www/js/util.js

```diff
@@ -1433,6 +1435,13 @@ function sendVideoUpdate() {
 
 /* chat */
 
+function stripImages(msg){
+    if (!USEROPTS.strip_image) {
+        return msg;
+    }
+    return msg.indexOf('<img') > -1 ? msg.replace(IMAGE_MATCH, "$1") : msg;
```

Comment:
Also it might be useful to replace the image with a clickable link instead of just URL text.